entrypoint

```diff
@@ -20,8 +20,7 @@ exec docker run -it --rm -v \$HOME:\$HOME -u \$(id -u) -w \$(pwd) -v /run/docker
 EOF
 elif [ "z$1" = "zsdkbase" ] ; then
     echo "Pulling and tagging the base image"
-    docker pull kbase/sdkbase
-    docker tag kbase/sdkbase kbase/kbase:sdkbase.latest
+    docker pull kbase/kbase:sdkbase2.latest
```

Comment:
is the tag line supposed to be deleted?

Why does the image name change other than appending a 2?

Reply:
Because the old image name seems to only have that name due to it being retagged and then pushed up. Since I just pushed kbase/kbase:sdkbase.latest without the rename, there isn't a reason to revert it to the old name.

There is kbase/sdkbase:develop up there now, but that is the "built from scratch" image, not the patched for python image built by kb_sdk/sdkbase/Dockerfile